lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -563,7 +562,100 @@ int amf_smf_send(
   }
   return rc;
 }
+/***************************************************************************
+**                                                                        **
+** Name:    amf_smf_validate_ie_params()                                  **
+**                                                                        **
+** Description: function to handle PDU Session Establishment Failures     **
+**                                                                        **
+**                                                                        **
+***************************************************************************/
+M5GSmCause amf_smf_validate_ie_params(
```

Comment:
This function is not validating all ies in message, just checking for session establishment failure cause. Can we instead call it `amf_smf_get_smcause()`

Reply:
Addressed review comment.